src/Entity/HealthCheckConfig.php

```diff
@@ -0,0 +1,114 @@
+<?php
+/**
+ * 2007-2019 PrestaShop and Contributors
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Academic Free License 3.0 (AFL-3.0)
+ * that is bundled with this package in the file LICENSE.txt.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/AFL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * @author    PrestaShop SA <contact@prestashop.com>
+ * @copyright 2007-2019 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/AFL-3.0 Academic Free License 3.0 (AFL-3.0)
+ * International Registered Trademark & Property of PrestaShop SA
+ */
+
+namespace PrestaShop\Module\HealthCheck\Entity;
+
+use Doctrine\ORM\Mapping as ORM;
+
+/**
+ * @ORM\Table()
+ * @ORM\Entity()
+ */
+class HealthCheckConfig
+{
+    /**
+     * @var int
+     *
+     * @ORM\Id
+     * @ORM\Column(name="id_health_check_config", type="integer")
+     * @ORM\GeneratedValue(strategy="AUTO")
+     */
+    private $id;
+
+    /**
+     * @var string
+     *
+     * @ORM\Column(name="token", type="string")
+     */
+    private $token;
+
+    /**
+     * @var string
+     *
+     * @ORM\Column(name="ips", type="string")
+     */
+    private $ips;
+
+    /**
+     * @return int
+     */
+    public function getId()
+    {
+        return $this->id;
+    }
+
+    /**
+     * @return string
+     */
+    public function getToken()
+    {
+        return $this->token;
+    }
+
+    /**
+     * @return HealthCheckConfig
+     */
+    public function setToken(string $token)
+    {
+        $this->token = $token;
+
+        return $this;
+    }
+
+    /**
+     * @return string
+     */
+    public function getIps()
+    {
+        return $this->ips;
+    }
+
+    /**
+     * @return HealthCheckConfig
+     */
+    public function setIps(string $ips)
+    {
+        $this->ips = $ips;
+
+        return $this;
+    }
+
+    /**
+     * Fill the object with given data. Data must be an array with this syntax:
+     * array(objProperty => value, objProperty2 => value, etc.).
+     *
+     * @return HealthCheckConfig
+     */
+    public function hydrate(array $data)
```

Comment:
```suggestion
    public function hydrate(array $data): self
```

Reply:
I always remove the return typehint when phpstorm generates it because I don't see it in the project
Will keep it now ;)